proto/astria/sequencer/v1alpha1/transaction.proto

```diff
@@ -102,3 +103,10 @@ message IbcHeight {
   uint64 revision_number = 1;
   uint64 revision_height = 2;
 }
+
+message IbcRelayerChangeAction {
+  // the address to update
+  bytes address = 1;
+  // set to true to add the address to the relayer list, false to remove it
+  bool addition = 2;
```

Comment:
Should we use an enum for this instead?

Reply:
up to you, might be a bit clearer/more self-documenting, i just went with a bool for simplicity for now

Reply:
okay, i'll change it then!

Reply:
@SuperFluffy @joroshiba updated to an enum